vivisect/analysis/generic/emucode.py

```diff
@@ -76,13 +76,6 @@ def prehook(self, emu, op, eip):
             emu.stopEmu()
 
         self.lastop = op
-        # Make sure we didn't run into any other
```

Comment:
love to better understand the logic of this cut.

Reply:
So it's to deal with the hydra function fun that's mentioned down below. 

I've hit this case a lot lately: we starting emulating another head of the function, run into the shared body, and then quit out due to the check here, never making the head of the new hydra function.

Reply:
Perhaps emulating all the way through isn't a perfect solution, but it does have the benefit of making sure everything is kosher and making sure the check in the `looksgood` function pass fine as well (for those hydra function heads that are short)